src/enhancer.js

```diff
@@ -0,0 +1,16 @@
+const enhancer = WrappedComponent => class Progress extends WrappedComponent {
+  componentDidUpdate() {
```

Comment:
I wanted to put the base` propTypes` and `defaultProps` in here but the linter cannot detect that there is a props validation for them